src/test/ui/rfc-2565-param-attrs/param-attrs-allowed.rs

```diff
@@ -5,83 +5,83 @@
 
 extern "C" {
     fn ffi(
-        #[allow(C)] a: i32,
+        #[allow(unused_mut)] a: i32,
```

Comment:
Lints from the `unused` group are `allow`ed by default in UI tests, so `allow` annotations in them don't test the normal behavior of `allow` unless there's a `#[warn(...)]` at the top of the file.

Reply:
Same in `param-attrs-feature-gate.rs`.